test/lib/bats.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Install Bash Automated Testing System
+
+set -o errexit
+
+export REPO_ROOT=$(git rev-parse --show-toplevel)
+BATS_VERSION=1.1.0
+
+if [[ "$1" ]]; then
+  BATS_VERSION=$1
+fi
+
+function installBats() {
+  bin=${REPO_ROOT}/bin
+  mkdir -p $bin
+  cd $bin
+  curl -sSL https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.tar.gz -o bats.tar.gz
+  tar -xvf bats.tar.gz --strip 1
```

Comment:
Maybe do:

```
mv $bin/bin/bats $bin/bats
```
here so that the path to `bats` executable isn't `bin/bin/bats`?

Reply:
bats depends on other scripts next to bin, I've moved them all to tmp.